Jellyfin.Server.Implementations/Users/UserManager.cs

```diff
@@ -315,7 +320,11 @@ public UserDto GetUserDto(User user, string? remoteEndPoint = null)
                     OrderedViews = user.GetPreferenceValues<Guid>(PreferenceKind.OrderedViews),
                     GroupedFolders = user.GetPreferenceValues<Guid>(PreferenceKind.GroupedFolders),
                     MyMediaExcludes = user.GetPreferenceValues<Guid>(PreferenceKind.MyMediaExcludes),
-                    LatestItemsExcludes = user.GetPreferenceValues<Guid>(PreferenceKind.LatestItemExcludes)
+                    LatestItemsExcludes = user.GetPreferenceValues<Guid>(PreferenceKind.LatestItemExcludes),
+                    CastReceiverId = string.IsNullOrEmpty(user.CastReceiverId)
+                        ? _serverConfigurationManager.Configuration.CastReceiverApplications.FirstOrDefault()?.Id
```

Comment:
Should probably put `_serverConfigurationManager.Configuration.CastReceiverApplications` in a variable. Pretty long lines.